src/templates/en/2019/chapter.html

```diff
@@ -1,26 +1,26 @@
-{{# IMPORTANT!
+{# IMPORTANT!
 
-- `chapter.html` is a "template for templates" used by the `generate_chapters.py` script, hence the strange template syntax (eg, double braces `{{% ... %}}`)
+- `chapter.html` is a "template for templates" used by the `generate_chapters.py` script, hence the strange template syntax (eg, double braces `{% ... %}`)
```

Comment:
```suggestion
- `chapter.html` is a "template for templates" used by the `generate_chapters.py` script
```

Reply:
I've updated the warning to include a bit about ejs, instead of removing it.